Description: re enable building the examples by default.
 Commit 2b397f985e4ef6ae5c0571a928bb1320fb048c61 made the examples
 not being made by default. The only way to get them is to
 enable -developer-build, which we do not want. So we simply
 patch out this commit.
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Forwarded: https://bugreports.qt-project.org/browse/QTBUG-30788

--- a/mkspecs/features/qt_example_installs.prf
+++ b/mkspecs/features/qt_example_installs.prf
@@ -92,12 +92,4 @@ probase = $$relative_path($$_PRO_FILE_PW
         allfiles -= doc
         !isEmpty(allfiles): warning("remaining files in $$_PRO_FILE_PWD_: $$allfiles")
     }
-
-    # Do not actually build the examples in production builds with -prefix
-    !equals(TEMPLATE, subdirs):prefix_build:!contains(QT_CONFIG, private_tests) {
-        TEMPLATE = aux
-        CONFIG -= have_target qt staticlib dll
-        SOURCES =
-        INSTALLS -= target
-    }
 }
